pages/api/templates/[...formID].ts

```diff
@@ -94,27 +84,23 @@ const templates = async (
 const templateCRUD = async ({
   ability,
   method,
-  user,
-  formID,
+  request,
   formConfig,
   isPublished,
   users,
 }: {
   ability: MongoAbility;
   method: string;
-  user: Session["user"];
-  formID?: string;
+  request: NextApiRequest;
   formConfig?: BetterOmit<FormRecord, "id" | "bearerToken">;
   isPublished: boolean;
   users: { id: string; action: "add" | "remove" }[];
 }) => {
+  const formID = request.query.formID ? (request.query.formID[0] as string) : undefined;
```

Comment:
If we change the route type from catch all to dynamic `[...formID]` to `[formID]` we can implicity type the `query.formID` as string.